db/views/hmis_households_v06.sql

```diff
@@ -0,0 +1,80 @@
+-- To update, run rails generate scenic:view hmis_households --replace
+WITH tmp1 AS (
+  (
+    SELECT
+      "Enrollment"."HouseholdID",
+      "Project"."ProjectID",
+      FALSE AS wip,
+      "Project"."data_source_id",
+      "Enrollment"."EntryDate",
+      "Exit"."ExitDate",
+      "Enrollment"."DateUpdated",
+      "Enrollment"."DateCreated"
+    FROM
+      "Enrollment"
+      LEFT OUTER JOIN "Exit" ON "Exit"."EnrollmentID" = "Enrollment"."EnrollmentID"
+      AND "Exit"."data_source_id" = "Enrollment"."data_source_id"
+      AND "Exit"."DateDeleted" IS NULL
+      JOIN "Project" ON "Project"."DateDeleted" IS NULL
+      AND "Project"."data_source_id" = "Enrollment"."data_source_id"
+      AND "Project"."ProjectID" = "Enrollment"."ProjectID"
+    WHERE
+      "Enrollment"."DateDeleted" IS NULL
+  )
+  UNION
+  ALL (
+    SELECT
+      "Enrollment"."HouseholdID",
+      "Project"."ProjectID",
+      TRUE AS wip,
+      "Project"."data_source_id",
+      "Enrollment"."EntryDate",
+      "Exit"."ExitDate",
+      "Enrollment"."DateUpdated",
+      "Enrollment"."DateCreated"
+    FROM
+      "Enrollment"
+      LEFT OUTER JOIN "Exit" ON "Exit"."EnrollmentID" = "Enrollment"."EnrollmentID"
+      AND "Exit"."data_source_id" = "Enrollment"."data_source_id"
+      AND "Exit"."DateDeleted" IS NULL
+      JOIN "hmis_wips" ON "hmis_wips"."source_id" = "Enrollment"."id"
+      AND "hmis_wips"."source_type" = 'Hmis::Hud::Enrollment'
+      JOIN "Project" ON "Project"."DateDeleted" IS NULL
+      AND "Project"."id" = "hmis_wips"."project_id"
+    WHERE
+      "Enrollment"."DateDeleted" IS NULL
+      AND "Enrollment"."ProjectID" IS NULL
```

Comment:
I spent a few minutes trying to see if I could reproduce a bug and create a regression test. I _think_ this line is the reason why the bug was not surfaced. `"Enrollment"."ProjectID"` is only NULL when the enrollment is WIP. When it's not, then even if the `hmis_wips` table still contains a now-deleted row for this enrollment, it wouldn't be returned by this query because non-WIPs are filtered out at this line.